packages/govern-console/src/containers/CreateDao/CreateDaoBasicInfo.tsx

```diff
@@ -45,9 +43,12 @@ const CreateDaoBasicInfo: React.FC<{
     isProxy,
   } = basicInfo;
 
-  useEffect(() => {
-    setValue('isExistingToken', isExistingToken);
-  }, [isExistingToken, setValue]);
+  const methods = useForm<ICreateDaoBasicInfo>({ defaultValues: { ...basicInfo } });
+  const { control, watch, getValues, trigger } = methods;
+
+  // useEffect(() => {
+  //   setValue('isExistingToken', isExistingToken);
+  // }, [isExistingToken, setValue]);
```

Comment:
if this useEffect() is no longer needed, can you remove it?

Reply:
done